### PR TITLE
updated directory structure

### PR DESCRIPTION
**Pull request by defalut**
updated for framework version 0.1.4

### CODE REVIEW

**Review by g-krisztian — COMMENTED** [review]
Please remove lein-template.iml.

**Review by ianffcs — APPROVED** [review]

File: resources/leiningen/new/xiana/src/backend/app/interceptors.clj [hunk 1]
```diff
@@ -0,0 +1,14 @@
+(ns interceptors
```
Comment (g-krisztian) [inline resources/leiningen/new/xiana/src/backend/app/interceptors.clj:1]: This file shouldn't be in interceptors folder?

**Review by g-krisztian — CHANGES_REQUESTED** [review]

File: resources/leiningen/new/xiana/src/backend/app/interceptors.clj (hunk 1 above)
Comment (defalut) [reply resources/leiningen/new/xiana/src/backend/app/interceptors.clj:1]: It follows the structure proposition https://chat.flexiana.com/#narrow/stream/89-open-source/topic/Directory.20structure

Thinking of that, the intent I see there is just to have var imports in this file and to have our own interceptor implementations in the interceptors/ folder. Is this right, @jiriknesl ?

If yes, then I will move the sample to some file in the interceptors/ folder

Comment (g-krisztian) [reply resources/leiningen/new/xiana/src/backend/app/interceptors.clj:1]: Ok, I've got it.

**Review by g-krisztian — APPROVED** [review]

**Pull request merged**